rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/ast/statements.kt

```diff
@@ -262,6 +262,21 @@ data class ZSubStmt(
         return emptyList()
     }
 }
+data class SubStmt(
```

Comment:
I think we should add a new blank line before SubStmt

Reply:
done